.github/workflows/main.yml

```diff
@@ -0,0 +1,120 @@
+name: GetFront RN SDK Testing Pipeline 🧪
+
+on: [push]
```

Comment:
Do we expect to run the testing pipeline on any push? I think it is enough to have it for PRs only.

Reply:
I think `pull_request` will not trigger, if after a PR is created and followup changes are committed

Reply:
The `pull_request` allows to trigger the pipeline for the branches which has opened the pull request. Not just regular push. This may help us to save recources.